lib/language_pack/ruby.rb

```diff
@@ -205,28 +205,6 @@ def warn_bundler_upgrade
     end
   end
 
-  # the base PATH environment variable to be used
-  # @return [String] the resulting PATH
-  def default_path(gem_layer_path = ".")
-    # Need to remove bin/ folder since it links
-    # to the wrong --prefix ruby binstubs
-    # breaking require. This only applies to Ruby 1.9.2 and 1.8.7.
-    #
-    # Because for 1.9.2 and 1.8.7 there is a "build" ruby and a non-"build" Ruby
-    paths = binstubs_relative_paths(gem_layer_path)
-    paths << "#{slug_vendor_jvm}/bin" if ruby_version.jruby?
-    paths << ENV["PATH"]
-    paths << "$HOME/bin"
-    paths << "/usr/local/bin:/usr/bin:/bin"
-    paths.join(":")
-  end
-
-  def binstubs_relative_paths(gem_layer_path = ".")
```

Comment:
With all the other changes I think it may be worth keeping this method around and changing the same and making the arg not have a default value.